pkg/manifests/manifests.go

```diff
@@ -329,7 +329,8 @@ var (
 )
 
 var (
-	ErrConfigValidation = fmt.Errorf("invalid value for config")
+	ErrConfigValidation       = fmt.Errorf("invalid value for config")
+	ErrUserWorkloadValidation = fmt.Errorf("invalid value for user-workload config")
```

Comment:
I don't think we need `ErrUserWorkloadValidation` in here, maybe I'm missing sth, but we don't have any check on it (not used) in the caller code.

(not related to the PR: same for `ErrConfigValidation` we only check it in the tests, I think we can get rid of it.)

Reply:
I removed it, I will takecare of ErrConfigValidation in separate PR :)